extensions/panache/mongodb-panache-common/runtime/src/main/java/io/quarkus/mongodb/panache/reactive/runtime/ReactiveMongoOperations.java

```diff
@@ -42,6 +42,11 @@
 public abstract class ReactiveMongoOperations<QueryType, UpdateType> {
     public final String ID = "_id";
     private static final Logger LOGGER = Logger.getLogger(ReactiveMongoOperations.class);
+
+    // update operators: https://docs.mongodb.com/manual/reference/operator/update/
+    private static final List<String> UPDATE_OPERATORS = Arrays.asList(
+            "$currentDate", "$inc", "$min", "$max", "$mul", "$rename", "$set", "$setOnInsert", "$unset");
```

Comment:
There are a lot more operators than this...

Reply:
Oh yes, I didn't scroll down the page :(
I added the missing operators, I didn't add `.$` as I'm afraid it could causes too much false positive.